### PR TITLE
Rollup improvements

### PR DESCRIPTION
**Pull request by chmelevskij**
* update rollup to latest
* update all plugins to latest
* enable sourcemaps
* use vendor.js file for all 3rd party scripts

https://user-images.githubusercontent.com/7774918/209908473-da431a6c-ae85-4546-a94c-fb0942e67f50.mov

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

File: gulpfile.js
Comment (haslinghuis) [inline gulpfile.js:442]: 👾

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
